src/USCC.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.26;
+
+import { USTB } from "src/USTB.sol";
+import { AllowList } from "src/AllowList.sol";
+
+contract USCC is USTB {
```

Comment:
pretty much just changing .isAllowed to .state1 in all functions that reference it